src/components/EthAddress.tsx

```diff
@@ -35,7 +35,8 @@ const Overrides = styled.div`
     color: ${COLORS.BLUE_GREY};
   }
 `;
-interface Props extends SProps {
+
+export interface Props extends SProps {
```

Comment:
I Found a way to avoid exporting component props using the react helper:
`React.ComponentProps<EthAddress>`
It avoids the extra export/import and so simplifies the naming of props since we no  longer  need to worry about collisions.